Google.Cloud.Compute.V1.AcceptanceTests/MainIntegrationTest.cs

```diff
@@ -24,22 +24,43 @@ public sealed class MainIntegrationTest
         private readonly AddressesClient _addrClient;
         private readonly RegionOperationsClient _ropsClient;
 
-        const string CredentialsPath = @"C:\Users\virost\AppData\Roaming\gcloud\legacy_credentials\virost@google.com\adc.json";
+        private const string TestCredentialsEnvironmentVariable = "GOOGLE_APPLICATION_CREDENTIALS";
+        private const string TestProjectEnvironmentVariable = "COMPUTE_TEST_PROJECT";
+        private const string TestRegionEnvironmentVariable = "COMPUTE_TEST_REGION";
 
         public MainIntegrationTest(ITestOutputHelper output)
         {
             _output = output;
-            _project = "client-debugging";
-            _region = "us-central1";
+            
+            // e.g. "client-debugging";
+            _project =  Environment.GetEnvironmentVariable(TestProjectEnvironmentVariable);
+            if (string.IsNullOrWhiteSpace(_project))
+            {
+                throw new ArgumentException($"Test project not set via the {TestProjectEnvironmentVariable} environment variable");
+            }
+
+            // e.g. "us-central1";
+            _region =  Environment.GetEnvironmentVariable("COMPUTE_TEST_REGION");
+            if (string.IsNullOrWhiteSpace(_region))
+            {
+                throw new ArgumentException($"Test region not set via the {TestRegionEnvironmentVariable} environment variable");
+            }
+
+            // e.g. "C:\Path\to\adc.json"
```

Comment:
(No change needed.) Will remove this when I move the code over - we assume ADC is sufficient, whether that's via a JSON file or not.